### ISSUE TITLE
Dark mode support

### ISSUE THREAD

**Issue by Supergamerx3000** [Daveiano/weewx-wdc#48]
Hello, will there be support for a dark mode in weewx-wdc? 
Can you adjust the colors by hand so that the skin is displayed in the dark?

Kind regards
Pascal

**Comment by Daveiano** [comment]
A dark mode is not yet supported but it would be a good addition to the skin!

Were you thinking about setting this via skin.conf (so the website is either in dark mode or in light mode), or providing a switch for the user like in the [Belchertown skin](https://belchertownweather.com/) (on the top right)?

**Comment by Daveiano** [comment]
@Supergamerx3000 Sorry, I changed my answer after you reacted.

This was more of a question, but I tend to add a Light/Dark Switch like in the [Belchertown skin](https://belchertownweather.com/)

**Comment by Supergamerx3000** [comment]
@Daveiano ah ok 😅 yes a light/dark switch sounds very good. 
I'm looking forward to the feature 👍

**Comment by Daveiano** [comment]
Here is the state of the current dark-mode development, feedback is appreciated!

![weewx-wdc-darkmode-gh-48](https://user-images.githubusercontent.com/3184897/183716265-657fd432-7f6b-4dad-ac6e-2d0320250165.png)

**Comment by Supergamerx3000** [comment]
wow the dark mode already looks very good.

The only thing that bothers me is the red diagrams, which are very difficult to see (maybe it's just me because of my red/green weakness) can you put a slightly lighter color in there?

Otherwise, I can't see anything else that could be improved on the dark mode, the bar on the left is also dark and the server information below.

great work 👍

**Comment by Daveiano** [comment]
@Supergamerx3000 The red diagrams were indeed difficult to see, the contrast between the red lines and the dark background was too low. I desaturated these vibrant red colours for the dark theme. 

The footer and the sidebar were actually not dark in the posted screenshots, this was on my to-do list. I updated this in the meantime and once this is released, these sections also will be dark. 

Dark mode will hit in 2.2.0, I will close this issue for now. Feel free to open another one once this is out if there are any problems regarding the accessibility.
